src/state/feed-feedback.tsx

```diff
@@ -28,9 +28,21 @@ import {useAgent} from './session'
 
 export const FEEDBACK_FEEDS = [...PROD_FEEDS, ...STAGING_FEEDS]
 
-export const DIRECT_FEEDBACK_INTERACTIONS = new Set<
+export const THIRD_PARTY_ALLOWED_INTERACTIONS = new Set<
   AppBskyFeedDefs.Interaction['event']
->(['app.bsky.feed.defs#requestLess', 'app.bsky.feed.defs#requestMore'])
+>([
+  // These are explicit actions and are therefore fine to send.
+  'app.bsky.feed.defs#requestLess',
+  'app.bsky.feed.defs#requestMore',
+  // These can be inferred from the firehose and are therefore fine to send.
+  'app.bsky.feed.defs#interactionLike',
+  'app.bsky.feed.defs#interactionQuote',
+  'app.bsky.feed.defs#interactionReply',
+  'app.bsky.feed.defs#interactionRepost',
```

Comment:
I added these for good measure because they're always obviously inferrable as public information, and correlating "this was served + there was an interaction" is practically close enough to just getting it directly. (But getting it directly is nicer for feeds that aren't tracking everything on the firehose.)